src/style.css

```diff
@@ -74,7 +74,7 @@
 }
 
 /* Reset buttons */
-.rdp-root button {
+.rdp-root :where(.rdp-day_button, .rdp-button_previous, .rdp-button_next) {
```

Comment:
I see what you mean, and I agree that could cause further confusion. Also I like a CSS only solution. We shouldn't use the `:where` pseudo selector, however, as I'm not sure anymore what will happen for Tailwind users there 🤷🏽 

Please could we [write everything twice](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself#WET) and reset each of the existing buttons?

```css
.rdp-day_button {
  /* existing ..rdp-day_button */
  /* (please remove overrides, if any, below)  */
  border: none;
  background: none;
  padding: 0;
  margin: 0;
  cursor: pointer;
  font: inherit;
  color: inherit;
}

.rdp-button_next,
.rdp-button_previous  {
  /* existing .rdp-button_next, .rdp-button_previous */
  /* (please remove overrides, if any, below)  */
  border: none;
  background: none;
  padding: 0;
  margin: 0;
  cursor: pointer;
  font: inherit;
  color: inherit;
}
```